src/Umbraco.AuthorizedServices/Services/IRefreshTokenParametersBuilder.cs

```diff
@@ -8,10 +8,18 @@ namespace Umbraco.AuthorizedServices.Services;
 public interface IRefreshTokenParametersBuilder
 {
     /// <summary>
-    /// Builds the the parameter dictionary used in token refresh requests.
+    /// Builds the parameter dictionary used in token refresh requests.
     /// </summary>
     /// <param name="serviceDetail">The service detail.</param>
     /// <param name="refreshToken">The refresh token.</param>
     /// <returns>A dictionary containing the refresh token parameters.</returns>
     Dictionary<string, string> BuildParameters(ServiceDetail serviceDetail, string refreshToken);
+
+    /// <summary>
+    /// Builds the parameter dictionary used in long lived token refresh requests.
+    /// </summary>
+    /// <param name="serviceDetail">The service detail.</param>
+    /// <param name="accessToken">The acccess token.</param>
+    /// <returns>A dictionary containing the refresh token parameters.</returns>
+    Dictionary<string, string> BuildParametesForOAuth2AccessTokenExchange(ServiceDetail serviceDetail, string accessToken);
```

Comment:
```suggestion
    Dictionary<string, string> BuildParametersForOAuth2AccessTokenExchange(ServiceDetail serviceDetail, string accessToken);
```

Reply:
Wondering also if this might be cleaner by introducing a new interface - `IExchangeTokenParametersBuilder`, rather than adding a new method to this interface?

Reply:
Moved to `IExchangeTokenParametersBuilder`.